.gitignore

```diff
@@ -328,3 +328,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+# Ignore .vscode files
+.vscode/
```

Comment:
Although this addition is absolutely motivated, let's try to keep the PRs focused on one subject at a time. You can always create more PRs with additional changes.

The fewer changes in one PR, the easier it is to go through and manage. Still, a PR needs to have enough changes to bring added value, so there's always a balancing game going on.

You don't have to change anything here. Just wanted to write this down to keep this in mind in the future.